utils_linux.go

```diff
@@ -123,11 +123,13 @@ func setupIO(process *libcontainer.Process, container *libcontainer.Container, c
 			}
 			uc, ok := conn.(*net.UnixConn)
 			if !ok {
+				conn.Close()
```

Comment:
I don't understand, if the result is not ok can the connection still be open. Are you sure?